app/__init__.py

```diff
@@ -0,0 +1,34 @@
+from flask import Flask, render_template, redirect, url_for, request
+from .forms.DeployForm import DeployForm
```

Comment:
One nice thing about SublimeText is that the Python plugin I'm using does pep8 checking. I'm guessing there's an equivalent for Atom somewhere!